app/Models/User.php

```diff
@@ -42,17 +42,17 @@ function roleUser(){
         return $this->hasMany(Role::class, 'user_id');
     }
     function imageUser(){
-        $this->hasOne(Image::class, 'imgable_id');
+        return $this->hasOne(Image::class, 'imgable_id');
     }
     function subjectUser(){
-        $this->belongsToMany(Subject::class, 'user_subject', 'user_id', 'subject_id');
+        return $this->belongsToMany(Subject::class, 'user_subject', 'user_id', 'subject_id');
     }
     function courseUser(){
-        $this->belongsToMany(Course::class, 'user_course', 'user_id', 'course_id');
+        return $this->belongsToMany(Course::class, 'user_course', 'user_id', 'course_id');
     }
     function taskUser()
     {
-        $this->belongsToMany(Task::class, 'user_task', 'user_id', 'task_id');
+        return $this->belongsToMany(Task::class, 'user_task', 'user_id', 'task_id');
     }
 
```

Comment:
đặt tên sai quy tắc 
https://laravel.com/docs/8.x/eloquent-relationships#one-to-many
function imageUser() {
sau () sẽ có 1 space nếu viết cùng dòng
và sử dụng đồng bộ 1 trong 2 kiểu viết